Tools/TransformTool/TransformTool.cs

```diff
@@ -240,6 +242,17 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 				if (manipulatorGameObject.activeSelf && (hoveringSelection || hasObject))
 					manipulatorGameObject.SetActive(false);
 
+				var hovers = new Dictionary<Transform, GameObject>(m_HoverGameObjects);
+				foreach (var kvp in hovers)
+				{
+					var rayOrigin = kvp.Key;
+					if (!directSelection.ContainsKey(rayOrigin))
```

Comment:
I made a note about this in my PR notes. I moved the selection back to the TransformTool because we would otherwise have to make SelectionTool aware of MiniWorlds.